### PR TITLE
BAU Remove 40% column width for the change link column and cleanup for styles, js files 

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/assets/stylesheets/partials/_gdsSummaryLists.scss [hunk 1]
```diff
@@ -3,10 +3,6 @@
 CSS Classes to support GDS Design System SummaryLists as implemented by the Exports Prototype
 ====================================
 */
-.govuk-summary-list__actions {
```
Comment (SteveSugden) [inline app/assets/stylesheets/partials/_gdsSummaryLists.scss:6]: The more of these we can remove the better (IMO).

Comment (SteveSugden) [reply app/assets/stylesheets/partials/_gdsSummaryLists.scss:6]: Should have asked before I approved.   How does this look on the summary page?

Comment (PatrykRudnicki) [reply app/assets/stylesheets/partials/_gdsSummaryLists.scss:6]: Yeah, that's true, I tried to remove more classes (please look at the next commit).

Comment (PatrykRudnicki) [reply app/assets/stylesheets/partials/_gdsSummaryLists.scss:6]: It looks similar like previously but there is more space for the first and second column. In practice the last column contains only link `Change` and had 40% width for it. This was the reason that there were a huge space between description and change link.

With this change, there is still some space between description and change link, but first column is wider and can display text without splitting it into few lines.

**Pull request merged**